lib/style_parser/filterer/index.js

```diff
@@ -19,11 +19,10 @@ var _ = require('underscore');
  * @param cssInput string
  */
 function filter(cssInput, cb) {
-
+  // Remove HTML comment start/end
+  cssInput = cssInput.replace('<!--', '').replace('-->', '');
```

Comment:
I think this should be moved into a function in this file with a comment that explains what it's doing and why (explaining these types of comments are not valid css comments, and css declarations are often wrapped in these comment tags).